<script>
try {
    //custom error
    throw new Error('oops');
}
catch (e) {

    console.log(e);
    //logMyErrors(e); 
}
</script>